pkg/worker/verifier/latest_value_map.go

```diff
@@ -11,22 +11,37 @@ import (
 )
 
 type LatestValueMap struct {
-	topic               string
-	LatestKvByPartition []map[string]string
+	topic string
+	// Latest offset for a given key
+	latestKoByPartition []map[string]int64
+	// Latest value for a given key
+	LatestKvByPartition []map[string][]byte
```

Comment:
Do we always insert into them at the same time? Maybe restructure as a map to a composite type?

Reply:
Currently yes, but only one variable is exported to JSON here (`LatestKvByPartition` is capitalized vs `latestKoByPartition`). I'd prefer to leave them separate for now.